django_saml2_auth/views.py

```diff
@@ -253,8 +270,16 @@ def signin(r):
 
     r.session['login_next_url'] = next_url
 
+    # Allow the requester to select the IDP they want to use. Required if multiple IDPs are configured.
+    selected_idp = r.GET.get('idp', None)
```

Comment:
Also what exactly does Wells Fargo need to do to get into the flow correctly? I ask since we need to specify this now as part of a questionnaire. Doest his meant he ACS request should be to

https://zx.zanbato.com/saml2_auth/acs?idp=wellsfargo

?

Reply:
Both good questions. Once I get a chance to sync up with Chris to set up a provider locally and examine the requests, I can figure out exactly how this should work.  Based off my rudimentary understanding, it would be `?idp={ENTITY_ID}` which should be a url I believe

Reply:
Okay, so answers:
BAML - Most recent commit adds a fallback, if no idp is specified, it defaults to the first in our list.

Wells Fargo - login url should be `https://zx.zanbato.com/saml2/login?idp=ENTITY_ID` where `ENTITY_ID` is the value they provide via metadata they send us. The rest should just work (based on testing with google SAML). The acs url is unchanged (no query parameters)